instagram/src/Components/Post/PostContainer/PostFooter/PostLikes/PostLikes.js

```diff
@@ -0,0 +1,47 @@
+import React, { Component } from 'react';
+
+import PostIcons from '../PostIcons/PostIcons';
+
+import './PostLikes.css';
+
+function numberWithCommas(x) {
+    return x.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ",");
+}
+
+class PostLikes extends Component {
+    constructor(props) {
+        super();
+
+        this.state = {
+            likes: props.likes,
+            liked: false
+        }
+    }
+
+    likePost = (e) => {
+        e.preventDefault();
+
+        if (this.state.liked) {
+            return this.setState({
+                likes: this.state.likes - 1,
+                liked: !this.state.liked
+            })
+        }
+
+        this.setState({
+            likes: this.state.likes + 1,
+            liked: !this.state.liked
+        })
```

Comment:
Why is your like functionality only incrementing by one?

Reply:
To give you the feel that you're logged in and can only like a post once.
First click you like and the hear becomes red. If you click it again it goes back to the default design and decrements by one